packages/react/src/components/Authenticator/shared/ConfirmationCodeInput.tsx

```diff
@@ -17,7 +18,9 @@ export const ConfirmationCodeInput = (
     label = `${translate('Code')} *`,
     placeholder = translate('Code'),
     required = true,
+    type = 'text',
   } = props;
+  // TODO: enforce type="number" on all confirmation codes
```

Comment:
Will change the default to `number` in another PR. This PR only makes `resetPassword` changes.

Reply:
Should we cut a separate ticket to track this?

Reply:
Will do, https://app.asana.com/0/1200334286823427/1201896333188125/f